yarn.lock

```diff
@@ -2827,23 +2827,7 @@ brace-expansion@^2.0.1:
   dependencies:
     balanced-match "^1.0.0"
 
-braces@^2.3.1, braces@^2.3.2:
```

Comment:
I checked out this branch and ran `yarn`, which resulted in some `yarn.lock` changes I think we'll want to commit.

(posted this comment to package.json by accident!)